GooglePlayInstant/Samples/SphereBlast/scripts/AnimateLoadingText.cs

```diff
@@ -0,0 +1,65 @@
+﻿// Copyright 2018 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class AnimateLoadingText : MonoBehaviour
+{
+	private UnityEngine.UI.Text loadingText;
+	private int timer = 0;
+	private int numCharacters;
+	private float deltaTimeSum = 0;
+	string[] loading = new string []
+	{
+		"L",
+		"O",
+		"A",
+		"D",
+		"I",
+		"N",
+		"G"
+	};
+
+	// Use this for initialization
+	void Start()
+	{
+		loadingText = GetComponent<UnityEngine.UI.Text>();
+		loadingText.text = loading[0];
+		numCharacters = loading.Length;
+	}
+	
+	// Update is called once per frame
+	void FixedUpdate()
+	{
+		deltaTimeSum += Time.deltaTime;
+		if ((int)deltaTimeSum == 1)
+		{
+			timer++;
+			deltaTimeSum = 0;
+			loadingText.text = ConstructString(timer);
+		}
+	}
+
+	string ConstructString(int index)
+	{
+		string myLoadingText = "";
+		for (int i = 0; i <= timer % numCharacters; i++)
```

Comment:
Delete numCharacters field and use loading.Length

Reply:
fixed